src/Evolution/Systems/GeneralizedHarmonic/Equations.hpp

```diff
@@ -80,4 +81,56 @@ struct ComputeDuDt {
       const tnsr::A<DataVector, Dim>& normal_spacetime_vector,
       const tnsr::a<DataVector, Dim>& normal_spacetime_one_form);
 };
+
+/*!
+ * \brief Compute the fluxes of the Generalized Harmonic formulation of
+ * Einstein's equations.
+ *
+ * \details The expressions for the fluxes is obtained
+ * from <a href="https://arxiv.org/abs/gr-qc/0512093">
+ * gr-qc/0512093 </a>.
+ * The fluxes for each variable are given by taking the principal part of
```

Comment:
I don't know what "principal part" means - according to the paper, these are the terms carrying the derivatives of the fields? Not asking you to change it just asking for my own clarification

Reply:
Yes, the parts of the equations that contains spatial derivatives of the fields. So the flux for each variable is determined solely by terms containing derivatives, the source terms don't contribute.